benchmark/common.js

```diff
@@ -88,52 +88,157 @@ Benchmark.prototype._queue = function(options) {
   return queue;
 };
 
-function hasWrk() {
-  const result = child_process.spawnSync('wrk', ['-h']);
-  if (result.error && result.error.code === 'ENOENT') {
-    console.error('Couldn\'t locate `wrk` which is needed for running ' +
-      'benchmarks. Check benchmark/README.md for further instructions.');
-    process.exit(1);
-  }
+function AutocannonBenchmarker() {
```

Comment:
Perhaps these abstractions should be moved to a separate file. I find this hard to read.